Documentation/In-depth/TheInstallTool/Index.rst

```diff
@@ -193,3 +193,17 @@ and displays hints if something goes wrong.
    might need to adjust the settings (via "Settings" > "Configuration Presets"
    > "Image Handling").
 
+After the last step of the installation you must verify the image handling settings
+under the configuration presets for Image Magick and Graphics Magick.
+E.g in Windows you must enter at the left of the button "Find executable in this directory" 
```

Comment:
Isn't it better to make a text-block for each of Windows and Linux / Mac?
"E.g" never shall be used anymore.

Reply:
You can add more lines here.

Reply:
I'd take the patch without describing unix - most unix system find it automatically and the way to configure it is basically the same on all systems (which Franz accurately described).